rct229/rulesets/ashrae9012019/ruleset_functions/get_swh_uses_associated_with_each_building_segment.py

```diff
@@ -18,11 +18,11 @@ def get_swh_uses_associated_with_each_building_segment(
     Returns
     -------
     swh_uses_list: list
-        A list containing the ids of all service water heating uses associated with a building segment
+        A list containing the distribution system ids of all service water heating uses associated with a building segment
     """
 
     swh_uses_list = find_all(
-        f'$.buildings[*].building_segments[*][?(@.id="{building_segment_id}")].zones[*].spaces[*].service_water_heating_uses[*].id',
+        f'$.buildings[*].building_segments[*][?(@.id="{building_segment_id}")].zones[*].spaces[*].service_water_heating_uses[*].served_by_distribution_system',
```

Comment:
I think this function is only getting `service_water_heating_uses` by a building segment or every segment. Either change the name of the function or update the logic to reflect **get_swh_uses**_associated_with_**each**_building_segment

Reply:
I'll change this based on the discussion above.

Reply:
As discussed, the function outcome has all the bldg_segment id as keys and their ```use``` values.